test/server/meta.integration.test.js

```diff
@@ -0,0 +1,64 @@
+/* @flow */
+
+import cheerio from 'cheerio';
+
+import { unpackSDKMeta } from '../../server';
+
+const sdkMetaList = [
```

Comment:
We can potentially simplify the building of theses so its easier to parse when reviewing tests. Right now we would have to really pause and read through each URL which can be difficult and potentially cause duplicate test cases or missed test cases in the long term. How do you feel about something like:
```
const params = [
  [
    {
      clientId: 'AQbghYd-7mRPyimEriYScIgTnYUsLnr5wVnPnmfPaSzwKrUe3qNzfEc5hXr9Ucf_JG_HFAZpJMJYXMuk',
      merchantId: 'K2US3FE3PQUEQ'
      intent: 'capture',
      lcaol: 'it_IT',
      components: ['buttons', 'funding-eligibility', 'messages']
      commit: false,
      currency: 'EUR'
    },
    {
      'data-partner-attribution-id': 'WooThemes_EC',
      'data-uid': 'uid_gwakesmdhfvakwylnuczrbplzdthxe',
    }
  ],
]

const buildTestUrl = ([queryParams, attrs]) => {
   // Build and return the url
  return `todo....`;
}

// $FlowIgnore[prop-missing] missing each property for test
test.each(params)('should unpack `sdkMeta` and load the script', ([queryParams, attrs]) => {
    const source = buildTestUrl([queryParams, attrs]);
    
    const { getSDKLoader } = unpackSDKMeta(Buffer.from(JSON.stringify(source))
        .toString('base64'));

    const $ = cheerio.load(getSDKLoader());
    const src = $('script').attr('src');

    if (source.url !== src) {
        throw new Error('The url should be the same');
    }
});
```

Reply:
doesn't have to be tuples but i jumped there. feel free to use objects of course.

Reply:
One could also make use of the test tables in jest but i kinda find those less readable than above to be honest

Reply:
That sounds good @westeezy, you're right it's a little hard to understand the object.